src/test/java/org/cactoos/io/AppendToTest.java

```diff
@@ -0,0 +1,109 @@
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.io;
+
+import java.io.File;
+import java.io.IOException;
+import org.cactoos.text.FormattedText;
+import org.cactoos.text.JoinedText;
+import org.cactoos.text.RandomText;
+import org.cactoos.text.TextOf;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.llorllale.cactoos.matchers.Assertion;
+import org.llorllale.cactoos.matchers.TextIs;
+import org.llorllale.cactoos.matchers.Throws;
+
+/**
+ * Test case for {@link AppendTo}.
+ *
+ * @since 1.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (100 lines)
+ */
+public final class AppendToTest {
+
+    /**
+     * Temporary files and folders generator.
+     */
+    @Rule
+    public final TemporaryFolder folder = new TemporaryFolder();
+
+    /**
+     * Ensures that AppendTo is failing on a negative predicate result.
+     * @throws Exception if fails
+     */
+    @Test
+    public void failsIfFileDoesNotExist() throws Exception {
+        // @checkstyle MagicNumber (1 line)
+        final File source = new File(new RandomText(5).asString());
+        new Assertion<>(
+            "Can't throw exception with proper message",
+            () -> () -> new AppendTo(source).stream(),
+            new Throws<>(
+                new FormattedText(
+                "Can't append to %s file. It does not exist",
+                    source.getAbsolutePath()
+                ).asString(),
+                IOException.class
+            )
+        ).affirm();
+    }
+
+    /**
+     * Ensures that AppendTo is appending to a given file.
+     * @throws Exception if fails
+     */
+    @Test
+    public void appendsToFile() throws Exception {
+        final File source = this.folder.newFile();
+        final String first = "abdcd";
+        new OutputTo(source).stream().write(first.getBytes());
+        final String second = "efgh";
+        new AppendTo(source).stream().write(second.getBytes());
+        new Assertion<>(
+            "Does not contain expected text",
+            () -> new TextOf(source),
+            new TextIs(new JoinedText("", first, second))
+        ).affirm();
+    }
+
+    /**
+     * Ensures that AppendTo is appending unicode text to a given file.
+     * @throws Exception if fails
+     */
+    @Test
+    public void appendsUnicodeToFile() throws Exception {
+        final File source = this.folder.newFile();
+        final String first = "\\u0026\\u2022";
```

Comment:
@Iprogrammerr Let's change these content to something more readble. Please, look at `TeeInputFromTextTest` for content examples.